docs/src/main/sphinx/release/release-462.md

```diff
@@ -0,0 +1,107 @@
+# Release 462 (16 Oct 2024)
+
+## General
+
+* Improve performance and memory consumption of the client protocol when
+  decoding data. ({issue}`23754`)
+
+
+
+
+## Database-based Resource Group Manager
```

Comment:
@martint @hashhar @piotrrzysko .. this are all the original release notes entries suggested in the PR. They should all go into the "General" section since we dont have a specific one for db resource group manager. Could you let me know if we really need all these four entries and with that much detail .. I will have to reword them all anyway to accommodate for the General section

Reply:
It's fine to have all those entries. They look like separate bugs.

Reply:
Yes, those were all separate bugs (which were fixed in single PR - but separate commits).

Looks good to me.